spk/comskip/Makefile

```diff
@@ -0,0 +1,38 @@
+SPK_NAME = comskip
+SPK_VERS = 0.81.089
+SPK_REV = 1
+SPK_ICON = src/comskip.gif
```

Comment:
Could you make this a PNG file please? And the file itself doesn't seem to be 256*256, but that would be the recommended size. I can't see the icon right now, but iirc, it was fairly low quality. Maybe take a look at that too.